### ISSUE TITLE
Some scenarios with `MenuBar` or `StatusBar` needs to fix Window position.

### ISSUE THREAD

**Issue by BDisp** [gui-cs/Terminal.Gui#3781]
The solution is define that it's mandatory to setup the views layout at design time. For now a `MenuBar` and `StatusBar` has a height of 1, but if it's changed in the future, then the `Window` must be layout with `Y = Pos.Bottom(menuBar)`, `Height = Dim.Fill(Dim.Func (() => statusBar.Frame.Height))`. In this case we need to change the current layout in the scenarios.

![image](https://github.com/user-attachments/assets/0627310b-8d5c-4c07-bd87-fc6fc59f9c47)

_Originally posted by @BDisp in https://github.com/gui-cs/Terminal.Gui/issues/3766#issuecomment-2397329331_

**Comment by tig** [comment]
Fixed in #3766

**Comment by BDisp** [comment]
> Fixed in #3766

All of them?

**Comment by BDisp** [comment]
I only opened this issue due to your comment here https://github.com/gui-cs/Terminal.Gui/pull/3766#issuecomment-2398851122.

**Comment by tig** [comment]
> > Fixed in #3766
> 
> All of them?

Yessir.

**Comment by tig** [comment]
> I only opened this issue due to your comment here [#3766 (comment)](https://github.com/gui-cs/Terminal.Gui/pull/3766#issuecomment-2398851122).

I appreciate it!

**Comment by BDisp** [comment]
> > > Fixed in #3766
> > 
> > 
> > All of them?
> 
> Yessir.

You didn't fix in the ASCIICustomButtonTest scenario.
Change this line:

![image](https://github.com/user-attachments/assets/31be6c7e-7ef5-4058-9722-bc40a99c83b1)

to:
```cs
_scrollViewTestWindow = new ScrollViewTestWindow { Y = Pos.Bottom (menu) };
```

Change this two lines:

![image](https://github.com/user-attachments/assets/efca535f-426f-4adc-ba53-8240d34ec8e3)

to:
```cs
        private FrameView _border = new ();
        private Label _fill = new ();

```

You also didn't fix the scenarios below.

ClassExplorer
ContextMenus
LineViewExample
Localization
SingleBackgroundWorker (The second MenuBar on StagingUIController class and set Modal = true)
WizardAsView

**Comment by tig** [comment]
You're gonna have to fix SingleBackgroundWorker. I can't figure out how its supposed to work. I've fixed the rest, I think.
